plugin/Renderer/InputFieldCheckboxRenderer.php

```diff
@@ -82,13 +86,16 @@ private function isMultipleSelect(string $key, FieldsCollection $pFieldsCollecti
 
 	private function buildFieldsCollection(): FieldsCollection
 	{
-		$pDIContainerBuilder = new ContainerBuilder;
-		$pDIContainerBuilder->addDefinitions(ONOFFICE_DI_CONFIG_PATH);
-		$pContainer = $pDIContainerBuilder->build();
+		if (empty($this->_pContainer)) {
```

Comment:
Building the container should be done in constructor, so that `if (empty())` can be removed